webpack.config.js

```diff
@@ -62,10 +61,7 @@ let config = dev.getDefaultWebpackConfig({
                 // getCoreNodeModule.js -> dist/node_modules/getCoreNodeModule.js
                 { from: './out/src/utils/getCoreNodeModule.js', to: 'node_modules' },
             ]
-        }),
-
-        // An instance of the StringReplacePlugin plugin must be present for it to work (its use is configured in modules).
-        new StringReplacePlugin()
```

Comment:
This was only used for Graph stuff and probably could've been removed as a part of this PR: 
https://github.com/microsoft/vscode-cosmosdb/pull/1362